### PR TITLE
Update deprecated client.NewEnvClient function

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

Due to [deprecation, `client.NewEnvClient` function](https://pkg.go.dev/github.com/docker/docker/client#pkg-index) has been updated to `NewClientWithOpts(FromEnv)`

## Type of change

- [x] Code cleanup/refactoring

# How Has This Been Tested?

**Test Configuration**:
* OS type & version: Ubuntu 20.04
* Hardware: x86-64
* Toolchain: Docker v19 and Go v1.16
* Edge Orchestration Release: v1.1.0

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: staticcheck.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 checks = ["all", "-ST1000", "-ST1016", "-U1000", "-S1000",
-	"-S1021", "-S1034", "-S1040", "-SA1019", "-SA4000"]
```
Comment (t25kim) [inline staticcheck.conf:2]: What's the reason for removing `SA1019`?

Comment (tdrozdovsky) [reply staticcheck.conf:2]: These changes fix the staticcheck warning, so now we can remove this exception [SA1019](https://staticcheck.io/docs/checks#SA1019),

Comment (t25kim) [reply staticcheck.conf:2]: I know what `SA1019` is. I wonder why it should be removed. Are you removing because the function does not work well?

Comment (t25kim) [reply staticcheck.conf:2]: I did not know that they are exception. I understand you want to activate the `SA1019` from staticcheck.

**Review by t25kim — APPROVED** [review]
LGTM

File: staticcheck.conf (hunk 1 above)
Comment (tdrozdovsky) [reply staticcheck.conf:2]: Сorrectly

**Review by suresh-lc — APPROVED** [review]
+1

**Review by MoonkiHong — APPROVED** [review]
+1

**Pull request merged**